packages/@okta/vuepress-theme-default/layouts/Layout.vue

```diff
@@ -37,7 +37,32 @@ export default {
     PromoBanner: () => import('../components/PromoBanner.vue')
   },
   mounted () {
-    window.addEventListener('scroll', this.onScroll)
+    window.addEventListener('load', () => {
+
+        let anchor = window.location.href.split('#')[1]
```

Comment:
Is `window.location.hash` no good?  or are you just trying to drop the '#'?

Reply:
Yup.  Better ways I'm sure, but this works for now?